docs/Salesforce-batchsource.md

```diff
@@ -163,6 +163,7 @@ Primary key (PK) Chunking splits query on large tables into chunks based on the
 
 Salesforce recommends that you enable PK chunking when querying tables with more than 10 million records or when a bulk query consistently times out. 
 However, the effectiveness of PK chunking depends on the specifics of the query and the queried data.
+We do not recommend enabling PK chunking when querying a large table and filtering out most of the data.
```

Comment:
Should also mention that each chunk counts as a separate query against the quota, so using PK chunking on a large table can end up counting as thousands of queries against the quota.

Reply:
thanks, done